### PR TITLE
[MIG] report_aeroo: Migration to 16.0

### PR DESCRIPTION
**Pull request by matiasperalta1**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/aeroo_reports/pull/48).

File: report_aeroo/demo/.~lock.template.odt# [hunk 1]
```diff
@@ -0,0 +1 @@
+,mnp,mnp-nb.adhoc,29.11.2022 17:13,file:///home/mnp/.config/libreoffice/4;
```
Comment (nicomacr) [inline report_aeroo/demo/.~lock.template.odt#]: This file must be erased.

**Review by nicomacr — CHANGES_REQUESTED** [review]
@matiasperalta1 Grate work, only one thing to change.
Read the comment below

File: report_aeroo/demo/.~lock.template.odt# (hunk 1 above)
Comment (matiasperalta1) [reply report_aeroo/demo/.~lock.template.odt#]: @nicomacr Ready

**Comment by nicomacr** [comment]
@roboadhoc r+

**Comment by roboadhoc** [comment]
@matiasperalta1 @nicomacr because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward

**Comment by nicomacr** [comment]
@roboadhoc rebase-ff

**Comment by roboadhoc** [comment]
Merge method set to rebase and fast-forward.

**Pull request merged**